doc/source/reference/commands/object_inspect.rst

```diff
@@ -747,6 +737,41 @@ The max value of each space is the max number of chunks. See
 :ref:`object-inspect-return-value-index-column-value-statistics-total-chunk-size`
 for the max number of chunks.
 
+.. _object-inspect-return-value-index-column-value-statistics-next-physical-segment-id:
+
+``INDEX_COLUMN_VALUE_STATISTICS_NEXT_PHYSICAL_SEGMENT_ID``
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. versionadded:: 9.0.2
+
+This value is the ID of the segment. The inspected index column use it as the next segment ID.
+The max value is the max number of segments. See
+:ref:`object-inspect-return-value-index-column-value-statistics-n-physical-segments`
+for the max number of segments.
+
+.. _object-inspect-return-value-index-column-value-statistics-n-physical-segments:
+
+``INDEX_COLUMN_VALUE_STATISTICS_N_PHYSICAL_SEGMENTS``
+"""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. versionadded:: 9.0.2
+
+This value the max number of segments. It depends on index size:
+
```

Comment:
It  is kind enough for users how to use this statistic result.
`If the number of segments tend to exceeds near the future, you need to consider to adopt ``INDEX_XXX`` flags.` and so on.